cmd/up/up.go

```diff
@@ -509,7 +512,39 @@ func (up *upContext) shutdown() {
 
 }
 
-func printDisplayContext(dev *model.Dev, divertURL string) {
+func (up *upContext) getUpEndpoints(ctx context.Context) []string {
```

Comment:
All the errors are ignored for the most part. I'm fine with returning an empty slice but should we log the error or are we relying on the original error being logged?